src/app/view/main/registration/registration-view.ts

```diff
@@ -2,53 +2,79 @@ import State from '../../../state/state';
 import { ElementCreator } from '../../../utils/element-creator';
 import InputFieldsCreator from '../../../utils/input-fields-creator';
 import { View } from '../../view';
-
-const CssClasses = {
-  REGISTRATION: 'registration',
-  TITLE: 'registration__title',
-  EMAIL: 'email',
-  PASSWORD: 'password',
-};
-
-const TEXT = {
-  FIELD_TEXT_ONE: 'Ваш мейл',
-  FIELD_TEXT_TWO: 'Ваш пароль',
-  TITLE: 'ЭТО СТРАНИЦА РЕГИСТРАЦИИ',
-};
-
-const KEY_FOR_SAVE = {
-  email: 'registration__email-input',
-  password: 'registration__password-input',
-};
+import { SIGN_UP_CLASSES, SIGN_UP_TEXT, SIGN_UP_KEY } from '../../../enums/enums';
 
 class RegistrationView extends View {
   state: State;
 
+  form: HTMLFormElement | ElementCreator;
+
   constructor(state: State) {
-    super('section', CssClasses.REGISTRATION);
+    super('section', SIGN_UP_CLASSES.REGISTRATION);
     this.state = state;
+    this.form = new ElementCreator('form', 'registration-form');
     this.configView();
   }
 
-  configView() {
-    const title = new ElementCreator('h1', CssClasses.TITLE, TEXT.TITLE);
+  // пофиксить длину функции, вынести типы в енум
+  // eslint-disable-next-line max-lines-per-function
```

Comment:
Wow... better change max-lines in eslint
And again, add TODO and do not forget about comments